src/shared/components/cells/CellCloneOverlay.tsx

```diff
@@ -0,0 +1,169 @@
+import React, {FC, useState, useContext, useEffect} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+
+// Context
+import {OverlayContext} from 'src/overlays/components/OverlayController'
+// Clockface
+import {
+  Dropdown,
+  Button,
+  Overlay,
+  ComponentColor,
+  InputLabel,
+  SlideToggle,
+  Form,
+} from '@influxdata/clockface'
+
+// Actions
+import {createCellWithView, deleteCellAndView} from 'src/cells/actions/thunks'
+import {getOverlayParams} from 'src/overlays/selectors'
+import {getDashboardIDs} from 'src/dashboards/utils/getDashboardIds'
+import {getOrg} from 'src/organizations/selectors'
+
+// Types
+import {AppState} from 'src/types'
+
+// Metrics
+import {event} from 'src/cloud/utils/reporting'
+import {Dashboard} from 'src/client'
+
+const CellCloneOverlay: FC = () => {
+  const [otherDashboards, setOtherDashboards] = useState<Dashboard[]>([])
+  const dispatch = useDispatch()
+  const {id: orgID} = useSelector(getOrg)
+
+  const currentDashboardID = useSelector(
+    (state: AppState) => state.currentDashboard.id
+  )
+
+  useEffect(() => {
+    getDashboardIDs(orgID).then(retrievedBoards => {
+      setOtherDashboards(
+        retrievedBoards.filter(board => board.id !== currentDashboardID)
+      )
+    })
+  }, [])
+
+  const {onClose} = useContext(OverlayContext)
+
+  const {view, cell} = useSelector(getOverlayParams)
+
+  const [destinationDashboardID, setDestinationDashboardID] = useState<string>(
+    ''
+  )
+  const [removeFromCurrentBoard, setRemoveFromCurrentBoard] = useState<boolean>(
+    false
+  )
+
+  const handleDeleteCell = (): void => {
+    dispatch(deleteCellAndView(currentDashboardID, cell.id, view.id))
+  }
+
+  const copyCellToDashboard = () => {
+    dispatch(
+      createCellWithView(
+        destinationDashboardID,
+        {...view, dashboardID: destinationDashboardID},
+        {...cell, dashboardID: destinationDashboardID},
+        null,
+        null,
+        true
+      )
+    )
+  }
+
+  const selectedDashboard = otherDashboards.find(
+    d => d.id === destinationDashboardID
+  )?.name
+
+  return (
+    <Overlay.Container maxWidth={400}>
+      <Overlay.Header title="Move Cell" onDismiss={onClose} />
+      <Overlay.Body className="cellCloneOverlayBody">
+        <Form.Element label="" className="otherDashboardDropdown">
+          <Dropdown
```

Comment:
nit: our dropdowns typically allow for resource creation, might be worth adding that here